arches/app/models/migrations/11579_ids_search_filter.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "10437_node_alias_not_null"),
```

Comment:
Looks like this dependency needs to be updated

Reply:
taking a look